.github/workflows/automerge-dependabot.yaml

```diff
@@ -8,7 +8,7 @@ permissions:
 jobs:
   dependabot:
     runs-on: ubuntu-latest
-    if: ${{ github.actor == 'dependabot[bot]' }}
```

Comment:
@vmanilo is this correct to merge or something you did to debug on your repo?

Reply:
yes, it's correct

the idea to not allow for dependabot automatically merge PR with dependecy updates on fork repo